draft-pauly-intarea-proxy-config-pvd.md

```diff
@@ -185,7 +210,16 @@ content-length = 222
   "identifier": "proxy.example.org.",
   "expires": "2023-06-23T06:00:00Z",
   "prefixes": [],
-  "proxies": ["https://proxy.example.org","https://proxy.example.org/masque{?target_host,target_port}"]
+  "proxies": [
+    {
+      "protocol": "connect",
+      "uri": "https://proxy.example.org"
```

Comment:
Unextended HTTP CONNECT doesn't have a scheme, only host and port

Reply:
I understand that you don't send a scheme in the request, but it seems like the proxy could still be identified with a scheme

Reply:
Well, pedantically there's no scheme though. I think the most correct way to do this would be to define "http-connect" and "https-connect" protocols and then have the uri only contain the host and optional port

Reply:
Otherwise we could keep the current URI but explain that only http and https are allowed and explain what it means

Reply:
Personally, I prefer http-connect and https-connect.
Maybe it would be clearer to use host name and port instead of URI for CNNECT and socks proxies.

Reply:
@ddragana updated!